<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoDialog</class>
 <widget class="QDialog" name="UserInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{background:transparent;}</string>
  </property>
  <layout class="QGridLayout" name="frameLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="layout">
     <item row="4" column="2" colspan="3">
      <widget class="QPlainTextEdit" name="addressPlainTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>116</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">width:50px;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">width:50px;</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>地址</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>UserInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
